src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -39,6 +39,10 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
   val workflowRoutes = queryRoute ~ workflowOutputsRoute ~ submitRoute ~ workflowStdoutStderrRoute ~ abortRoute ~
     callOutputsRoute ~ callStdoutStderrRoute ~ validateRoute ~ metadataRoute
 
+  def docsRoute = path("swagger" / "cromwell.yaml") {
```

Comment:
We used to server all of the static `swagger-ui` files, along with json generated by `spray-swagger`. Now we just need this one static yaml.